File: Python/448.FindAllNumbersDisappearedinanArray.py
class Solution(object):
    def findDisappearedNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        ans=list()
        for i in range(0,len(nums)):
            tmp=abs(nums[i])-1
            if nums[tmp]>=0:
                nums[tmp]=-nums[tmp]
        for i in range(0,len(nums)):
            if nums[i]-1>=0:
                ans.append(i+1)
        return ans
        

File: C++/448. Find_All_Numbers_Disappeared_in_an_Array.cpp
//Assigned #1016 
//CODE

class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        set<int,greater<int>> s;
        vector<int> v;//return result vector
        for(int i=0;i<nums.size();i++)
        {
            s.insert(nums[i]);//insert in a set for distinct condition
        }
        for(int i=1;i<=nums.size();i++)
        {
            if(s.count(i)==0)
            {
                v.push_back(i);//push the numbers with zero frequency
            }
        }
        return v;
        
    }
};
